app/components/trainees/confirmation/confirm_publish_course/view.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1 class="govuk-heading-l"><%= heading %></h1>
```

Comment:
Can we add all this content to i18n instead of hard coding it?

Reply:
Sure, heading is already translated in view.rb object.